node/src/main/resources/postgresql/node/migration/V3__peer_table.sql

```diff
@@ -1 +1 @@
-create table if not exists "peers" ("address" VARCHAR NOT NULL PRIMARY KEY UNIQUE,"port" INT NOT NULL,"last_seen" TIMESTAMP,"first_seen" TIMESTAMP, "network_id" INT)
+create table if not exists "peers" ("address" VARCHAR NOT NULL, "port" INT NOT NULL, "last_seen" TIMESTAMP,"first_seen" TIMESTAMP, "network_id" INT, "service_bytes" VARCHAR, constraint pk_peers PRIMARY KEY (address,port));
```

Comment:
Now treats peers with same address but different ports as distinct. Needed for multiple nodes on localhost with different ports.

Reply:
Won't this break migrations? I believe we have to add a new migration called `V4` and then add the new `service_bytes` column. Modifying existing deployed migrations will cause an exception on startup if the user had started with the `V3__peer_table.sql` migration.